### DISCUSSION TITLE
[Thesis] Webschnittstelle

### DISCUSSION THREAD

**Discussion by AMatutat** [Programmiermethoden/Dungeon#1376] [Ideas]
Zu Konzeptionieren und Implementieren ist eine Webschnittstelle (REST? Was besseres?)
mit dessen Hilfe zu einem von außen Informationen in das Dungeon gesendet werden können (hier wäre auch zu Überlegen, was man damit triggert) als auch Informationen aus dem Dungeon nach außen sendet. 

Ist eigentlich für die Roboter gedacht, könnte aber auch gut in Verbindung mit #302 sinnvoll sein

**Comment by cagix** [comment]
mit #2109 erstmal obsolet.
